### ISSUE TITLE
Can not add or clear the priority icon

### ISSUE THREAD

**Issue by wangzhaohe** [leo-editor/leo-editor#2870]
Leo Log Window
Leo 6.7.0-devel, devel branch, build fae87f5801
2022-09-17 19:12:38 +0800
Python 3.9.5, PyQt version 5.15.2
darwin

<img width="694" alt="image" src="https://user-images.githubusercontent.com/2628013/190862995-9d806885-65e5-4b46-a855-b8875a6fc70b.png">

**Comment by edreamleo** [comment]
@wangzhaohe Thanks for your testing!

**Comment by edreamleo** [comment]
@wangzhaohe Oh my. Git lists only the *first* few digits of the sha hash, but Leo's signon lists the *last* few digits of the signon. I haven't yet discovered how to recover the hash given the *last* digits. I'll create a separate issue to make the "build" info more useful.

**Comment by edreamleo** [comment]
@wangzhaohe I do see that the todo icons are missing. A git bisect will discover the culprit :-)

**Comment by edreamleo** [comment]
git bisect reports:

```console
3132cdcb3915655853e590f6b38d67be530a7cd6 is the first bad commit
commit 3132cdcb3915655853e590f6b38d67be530a7cd6
Author: Edward K. Ream <edreamleo@gmail.com>
Date:   Mon Sep 5 05:50:44 2022 -0500

    v.setDirty calls v.updateIcon. Three methods take vnode arg instead of position arg
```

**Comment by wangzhaohe** [comment]
@edreamleo Sorry! The build is 9dcbff603a.

Build fae87f5801 is my local commit hash. The reason is that I have changed the `qt_frame.py` a little then with a merge, so the hash changed. I forgot it. I make you an apology.

The changed content is line `4769 r = max(0.0, f.secondary_ratio - 0.1) --> 0.4`. So I can expand-log-pane one time with the ratio 0.4, not several times with 0.1. It only for my convenient.

**Comment by edreamleo** [comment]
@wangzhaohe Thanks for your comment. git bisect put me on the right track :-)

**Comment by edreamleo** [comment]
Fixed via PR #2871.
